pos_invoicing/models/account_invoice.py

```diff
@@ -21,11 +21,10 @@ class AccountInvoice(models.Model):
     @api.multi
     def action_cancel(self):
         self._check_pos_pending_payment()
-        return super(AccountInvoice, self).action_cancel()
+        return super().action_cancel()
 
-    @api.multi
+    @api.one
```

Comment:
Isn't this deprecated since 9.0?

Reply:
No it is used on odoo12. See : 
https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L138
and the current implementation generate the bug #644. 

``@api.multi + self.ensure_one()`` can not overload ``@api.one``.

Reply:
Ok, ok.